examples/205-automl.ipynb

```diff
@@ -25,6 +25,7 @@
     "- [General AutoML](#chapter_2)\n",
```

Comment:
<p>It seems like you should also update the file: <a href="https://github.com/etna-team/etna/tree/master/examples#automl" target="_blank">https://github.com/etna-team/etna/tree/master/examples#automl</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>